src/gui/editors/AutomationEditor.cpp

```diff
@@ -1587,12 +1587,36 @@ void AutomationEditor::drawLevelTick(QPainter & p, int tick, float value)
 
 		p.fillRect( x, y_start, rect_width, rect_height, currentColor );
 	}
-
+#ifdef LMMS_DEBUG
 	else
 	{
 		printf("not in range\n");
 	}
+#endif
+}
+
+
+
 
+// center the vertical scroll position on the first object's value
+void AutomationEditor::centerTopBottomScroll()
+{
+	// default to the m_scrollLevel position
+	int pos = (int) m_scrollLevel;
+	// If a pattern exists...
+	if (m_pattern) {
```

Comment:
The brace should be in the next line.

Reply:
Oof, that's on me. Will update and the other.